### PR TITLE
Sentry cleanup, part 1.

### PR DESCRIPTION
**Pull request by mayhem**
I'm going to do a weekly PR to help cleanup useless sentry messages -- we have millions of them and a lot of them don't need to be there.

This first PR removes logging from 400 errors for listen validation, where clients botch a request -- there is no reason for us to log these.

### CODE REVIEW

**Comment by mayhem** [comment]
And I've changed the "inactive user" message from error to debug.

File: listenbrainz/webserver/views/api_tools.py [hunk 1]
```diff
@@ -114,42 +114,43 @@ def validate_listen(listen, listen_type):
...
+            if 'listened_at' in listen:
+                raise APIBadRequest("JSON document must not contain listened_at while submitting "
+                                    "playing_now.", listen)
 
-        if 'track_metadata' in listen and len(listen) > 1:
-            log_raise_400("JSON document may only contain track_metadata as top level "
-                           "key when submitting now_playing.", listen)
+            if 'track_metadata' in listen and len(listen) > 1:
+                raise APIBadRequest("JSON document may only contain track_metadata as top level "
+                                    "key when submitting now_playing.", listen)
```
Comment (alastair) [inline listenbrainz/webserver/views/api_tools.py:142]: can you open a ticket for this or fix to be consistent with the docs - one error message says now_playing and the next one down says playing now

File: listenbrainz/webserver/views/api_tools.py [hunk 2]
```diff
@@ -114,42 +114,43 @@ def validate_listen(listen, listen_type):
...
         if not listen['track_metadata']['artist_name']:
-            log_raise_400("JSON document does not contain required "
-                           "track_metadata.artist_name.", listen)
+            raise APIBadRequest("JSON document does not contain required "
+                                "track_metadata.artist_name.", listen)
         if not isinstance(listen['track_metadata']['artist_name'], str):
-            log_raise_400("artist_name must be a single string.", listen)
+            raise APIBadRequest("artist_name must be a single string.", listen)
     except KeyError:
         log_raise_400("JSON document does not contain a valid metadata.track_name "
```
Comment (alastair) [inline listenbrainz/webserver/views/api_tools.py:155]: is it intentional that this one isn't updated?

**Review by alastair — APPROVED** [review]
Once my two questions are answered, this looks fine

**Comment by mayhem** [comment]
OK, I'm considering this done. I'll merge and see if the JS test failure is spurious.

**Pull request merged**